src/themes/circuit.js

```diff
@@ -205,6 +205,13 @@ export const grid = {
     cols: 12,
     maxWidth: '880px',
     gutter: spacings.giga
+  },
+  afterTera: {
```

Comment:
Can we keep this as `tera`? All the other grid configs have the name of the breakpoint specifying their `minWidth`. And since you are not using `afterTera` here, it would make sense to me to use `tera` for the grid.

Reply:
We can't without extra changes. The key (`afterTera`) here is the `mq.*` which will be used for the css generation.

Take a look here:
https://github.com/sumup/circuit-ui/blob/master/src/components/Grid/Grid.js#L10

Also, it is not visible in the changes, but the first one is actually `untilKilo`, so it is not inconsistent in this case.